zephyr/test/vmh.c

```diff
@@ -1,12 +1,13 @@
 // SPDX-License-Identifier: BSD-3-Clause
 /*
- * Copyright(c) 2023 Intel Corporation. All rights reserved.
- *
- * Author: Guennadi Liakhovetski <guennadi.liakhovetski@linux.intel.com>
- */
+* Copyright(c) 2023 Intel Corporation. All rights reserved.
+*
+* Author: Guennadi Liakhovetski <guennadi.liakhovetski@linux.intel.com>
+*/
```

Comment:
the indentation was correct, no need to change

Reply:
clang formater automatically changed it - reverting